charts/sftd/values.yaml

```diff
@@ -81,3 +81,5 @@ joinCall:
     # Overrides the image tag whose default is the chart appVersion.
     tag: "1.19.5"
 
+# Make the SFT to proxy itself with some TURN server when trying to connect to clients
```

Comment:
I'm not sure I fully understand everything, so please get confirmation from AVS. But to my understanding, the following might be a more helpful comment:

```suggestion
# When enabled, allow media traffic to flow clients <--> TURN <--> SFT [that allows UDP OR TCP OR TLS between clients and TURN. UDP traffic is still required between TURN and SFT]
# When disabled, only allow media traffic to flow clients <--> SFT [that requires direct UDP packets between clients and SFT]
```

Reply:
Admittedly, the comment is phrased in a weird way. but your suggestion is not true. When setting the flag, it would actually be: clients <--> TURN <--> TURN <--> SFT (with the arrows optionally being TCP-only). See [this PR #146 at wire-docs](https://github.com/wireapp/wire-docs/pull/146/files#diff-3131d91b6a6d39e99b97c9303ac446d36e6d948fd4b2de09f76bf2e919dcbae3R73-R76). I'm still failing to find some better phrasing.

Reply:
Okay sorry, I don't fully understand. Are you able to maybe partially re-use some of the phrasing and replace some of the arrows in question with better arrows?
Or perhaps link to the section in the docs that explain what happen with this is enabled or disabled?

Reply:
I made another small change to the comment in the code, which also refers to a [recent change in the docs](https://github.com/wireapp/wire-docs/pull/146). Would you mind taking another quick look and let me know if there is still sth to be improved.